src/Platform/Microsoft.Testing.Platform/OutputDevice/Console/LoggerArtifact.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Testing.Platform.OutputDevice.Console;
+
+/// <summary>
+/// An artifact / attachment that was reported during run.
+/// </summary>
+internal record class LoggerArtifact(bool OutOfProcess, string? Assembly, string? TargetFramework, string? Architecture, string? TestName, string Path);
```

Comment:
What about `TestRunArtifact`? In the core platform, we make the difference between an artifact attached to a test and a "global" artifact, it might be interesting to keep the same logic.

Reply:
Renamed.